Framework/PythonInterface/plugins/algorithms/SNSPowderReduction.py

```diff
@@ -1398,7 +1413,28 @@ def _process_container_runs(self, can_run_numbers, samRunIndex, preserveEvents,
         :param can_run_numbers:
         :return:
         """
-        can_run_ws_name, can_run_number = self._generate_container_run_name(can_run_numbers, samRunIndex)
+        if self._enableInterpo:
+            can_run_ws_name_1, can_run_number_1 = self._generate_container_run_name(can_run_numbers, 0)
+            can_run_ws_name_2, can_run_number_2 = self._generate_container_run_name(can_run_numbers, 1)
+
+            self._focusAndSum([can_run_number_1], preserveEvents, final_name=can_run_ws_name_1, absorptionWksp=absorptionWksp)
+            self._focusAndSum([can_run_number_2], preserveEvents, final_name=can_run_ws_name_2, absorptionWksp=absorptionWksp)
+            empty_run_ws_group = api.GroupWorkspaces([can_run_ws_name_1, can_run_ws_name_2])
+            interpo_ws = api.InterpolateBackground(empty_run_ws_group, self._interpoTemp, OutputWorkspace="InterpolatedBackground")
+            # smooth background
+            smoothParams = self.getProperty("BackgroundSmoothParams").value
+            if smoothParams is not None and len(smoothParams) > 0:
+                api.FFTSmooth(
+                    InputWorkspace=interpo_ws,
+                    OutputWorkspace=interpo_ws,
+                    Filter="Butterworth",
+                    Params=smoothParams,
+                    IgnoreXBins=True,
+                    AllSpectra=True,
+                )
+            return interpo_ws
+        else:
+            can_run_ws_name, can_run_number = self._generate_container_run_name(can_run_numbers, samRunIndex)
```

Comment:
Are there any tests of correct performance of this new behavior?

Reply:
This is one small portion of a larger algorithm for reduction. I was having a hard time coming up with test to specifically check this functionality

Reply:
The new functionality here is only an `if` clause.  The behavior inside the `if` was completed inside PR #35977 , and has associated tests.